source/includes/steps-install.yaml

```diff
@@ -0,0 +1,67 @@
+title: "Install Xcode from the App Store."
+ref: install-xcode
+content: |
+  
+  Install Xcode -- it takes awhile. In the meantime, you can follow the rest of the steps.
```

Comment:
Install Xcode. This installation may take a while, so proceed to the following steps while Xcode is installing.

or something like that.